usr/share/rear/layout/prepare/GNU/Linux/133_include_mount_filesystem_code.sh

```diff
@@ -135,6 +135,13 @@ mount_fs() {
             echo "mount $device $TARGET_FS_ROOT$mountpoint"
             ) >> "$LAYOUT_CODE"
             ;;
+        (ext2 | ext3 | ext4)
+            (
+            echo "mkdir -p $TARGET_FS_ROOT$mountpoint"
+            echo "mount $mountopts $device $TARGET_FS_ROOT$mountpoint"
+            echo "mount $mountopts,remount,user_xattr $device $TARGET_FS_ROOT$mountpoint"
```

Comment:
I cannot see what the reason is for the additional
```
echo "mount $mountopts,remount,user_xattr $device $TARGET_FS_ROOT$mountpoint"
```
that is only done for ext2/3/4
so please asdd an explanatory comment what the resoning behind is
in particular why this is done in any case (and not only in case of IBM Z)
but only for ext2/3/4 and why it is not needed for other filesytems
cf. "Code must be easy to understand" in
https://github.com/rear/rear/wiki/Coding-Style

Reply:
i will check this against intel as to why it applies to s390